code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -93,7 +93,7 @@ God bless America.
 	else if(default_deconstruction_screwdriver(user, "fryer_off", "fryer_off" ,I))	//where's the open maint panel icon?!
 		return
 	else
-		if(is_type_in_typecache(I, deepfry_blacklisted_items))
+		if(is_type_in_typecache(I, deepfry_blacklisted_items) || (I.flags_1 & (ABSTRACT_1 | NODROP_1 | DROPDEL_1)))
```

Comment:
isn't nodrop already handled on the next if? also this if feels wrong

Reply:
Not entirely sure since doUnEquip (which is what handles nodrop) checks for !no_move or !force, and not just straight no_drop